rpc/src/v1/types/confirmations.rs

```diff
@@ -94,6 +105,26 @@ impl fmt::Display for SignRequest {
 	}
 }
 
+impl From<(H160, Bytes)> for EthSignRequest {
+	fn from(tuple: (H160, Bytes)) -> Self {
+		EthSignRequest {
+			address: tuple.0,
+			data: tuple.1,
+		}
+	}
+}
+
+impl fmt::Display for EthSignRequest {
+	fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+		write!(
+			f,
+			"sign 0x{} with {}",
```

Comment:
Should we change the previous implementation to distinguish between eip191 and this?